### PR TITLE
[REEF-1007] Remove invalid notification email from Travis CI build script

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
JIRA:
  [REEF-1007](https://issues.apache.org/jira/browse/REEF-1007)

Pull Request:
  This closes #

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ cache:
 
 notifications:
```
Comment (tcNickolas) [inline .travis.yml]: Is notifications section mandatory if we don't want any notifications? Maybe we could remove it at all?

Comment (dongjoon-hyun) [inline .travis.yml]: Good point. I think you're right, but let me check for a while.

Comment (dongjoon-hyun) [inline .travis.yml]: Oh, according to the documentation (https://docs.travis-ci.com/user/notifications/), the default is sending notifications to committers. To disable notification, we should use `false` explicitly.

```
By default, email notifications are sent to the committer and the commit author, if they are members of the repository (that is, they have push or admin permissions for public repositories, or if they have pull, push or admin permissions for private repositories).
```

Comment (tcNickolas) [inline .travis.yml]: Thank you for checking. I'll merge

**Comment by dongjoon-hyun** [comment]
Thank you for merging and amending commit log, @tcNickolas .

**Comment by dongjoon-hyun** [comment]
Oh, @tcNickolas . Did you get the notification message? I suddenly got one. It should be like the following.
```
notifications:
  email: false
```

**Comment by tcNickolas** [comment]
No, I didn't.
We probably can fix it under the same JIRA number.

**Comment by dongjoon-hyun** [comment]
Sure! I reopen the issue and made a quick fix PR, #680 .

**Pull request closed without merging**